providers/src/airflow/providers/fab/version_compat.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
No need to add that. FAB in main is already Airflow 3.0+

Reply:
Thanks! I thought it could impact that `fab` too, reverting that now

Reply:
Removed them from the PR, thanks again!

Reply:
Removed them from the PR, thanks again!